DataConnectors/Qualys VM/azuredeploy_QualysVM_API_FunctionApp.json

```diff
@@ -24,7 +24,11 @@
           },
           "Uri": {
               "type": "string",
-              "defaultValue": "https://<API Server URL>/api/2.0/fo/asset/host/vm/detection/?action=list&detection_processed_after="
+              "defaultValue": "https://<API Server URL>/api/2.0/fo/asset/host/vm/detection/?action=list&vm_processed_after="
+          },
+          "FilterParameters": {
+              "type": "string",
+              "defaultValue": "&truncation_limit=1000"
```

Comment:
How can the default value of 1000 be overwritten if it's passed in the filter parameters?

Reply:
They simple just change the 1000 to whatever value they would like to before clicking Purchase (Deploy)